src/connectors/search-box/connectSearchBox.js

```diff
@@ -35,6 +35,8 @@ Full documentation available at https://community.algolia.com/instantsearch.js/c
  * @property {function(string)} refine Sets a new query and searches.
  * @property {function()} clear Remove the query and perform search.
  * @property {Object} widgetParams All original `CustomSearchBoxWidgetOptions` forwarded to the `renderFn`.
+ * @property {boolean} isSearchStalled `true` if the search results takes more than a certain time to come back
```

Comment:
Should we document "a certain time"? I would be curious as a dev to know more. This may be too vague

Reply:
indeed there is a default value. Will doc that.

Reply:
Done (only happen a few lines after :))